nltk/test/unit/translate/test_bleu.py

```diff
@@ -22,8 +22,10 @@ def test__modified_precision(self):
         references = [ref1, ref2] 
         
         # Testing modified unigram precision.
-        hyp1_unigram_precision =  _modified_precision(references, hyp1, n=1) 
+        hyp1_unigram_precision =  _modified_precision(references, hyp1, n=1)
         assert (round(hyp1_unigram_precision, 4) == 0.2857)
+        # With assertAlmostEqual at 4 place precision.
+        self.assertAlmostEqual(hyp1_unigram_precision, 0.2857142857142857, places=4)
```

Comment:
I like the use of `assertAlmostEqual(a, b, 4)`, but is there a need to keep the test for `assert(round(a, 4), b)`?